packages/web-components/playwright.config.ts

```diff
@@ -0,0 +1,25 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  projects: [{ name: 'chromium' }, { name: 'firefox' }, { name: 'webkit' }],
+  reporter: 'list',
+  testMatch: /.*\.spec\.ts$/,
+  retries: 3,
+  fullyParallel: process.env.CI ? false : true,
+  timeout: process.env.CI ? 10000 : 30000,
+  use: {
+    baseURL: 'http://localhost:6006/iframe.html',
+    viewport: {
+      height: 1280,
+      width: 720,
+    },
+  },
+  webServer: {
+    // double-quotes are required for Windows
+    command: `node -e "import('express').then(({ default: e }) => e().use(e.static('./storybook-static')).listen(6006))"`,
```

Comment:
What builds the storybook to `./storybook-static`?

Reply:
Locally you'd run the storybook server and then run `yarn test`. I added a note in the package readme about this. In the case of CI, it'll run the server itself, but locally we reuse it.